app/src/test/java/org/simple/clinic/search/PatientSearchScreenControllerTest.kt

```diff
@@ -5,118 +5,44 @@ import com.nhaarman.mockito_kotlin.mock
 import com.nhaarman.mockito_kotlin.never
 import com.nhaarman.mockito_kotlin.times
 import com.nhaarman.mockito_kotlin.verify
-import com.nhaarman.mockito_kotlin.whenever
 import io.reactivex.subjects.PublishSubject
 import org.junit.Before
 import org.junit.Test
-import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthAndAgeVisibility
-import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthFormatValidator
-import org.simple.clinic.widgets.ageanddateofbirth.DateOfBirthFormatValidator.Result
 import org.simple.clinic.patient.PatientRepository
 import org.simple.clinic.widgets.UiEvent
 
 class PatientSearchScreenControllerTest {
 
   private val screen: PatientSearchScreen = mock()
   private val repository: PatientRepository = mock()
-  private val dateValidator: DateOfBirthFormatValidator = mock()
 
   private lateinit var controller: PatientSearchScreenController
   private val uiEvents = PublishSubject.create<UiEvent>()
 
   @Before
   fun setUp() {
-    controller = PatientSearchScreenController(repository, dateValidator)
+    controller = PatientSearchScreenController(repository)
 
     uiEvents.compose(controller).subscribe { uiChange -> uiChange(screen) }
   }
 
   @Test
-  fun `search button should remain enabled only when both name and age or date of birth are present`() {
+  fun `search button should remain enabled only when both name is present`() {
```

Comment:
`both` can also be removed from this function's name

Reply:
Done.